src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+  let object = state;
```

Comment:
зараз в object потрапляє силка на обєкт state. тобі потрібно зробити його копію. І таким чином ти позбудешся непотрібник констант і Object.asign в кожному випадку

```suggestion
  let object = { ...state };
```